.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
   - BUILDTAGS='btrfs_noversion libdm_no_deferred_remove containers_image_openpgp'
 
 script: >
-  sudo docker run --privileged -ti --rm --user $(id -u):$(id -g)
+  sudo docker run -ti --rm --user $(id -u):$(id -g)
```

Comment:
I think we need the --privileged to get the tests green.

Reply:
The failures are just in creating keyrings.  The rest of the tests are passing, as far as I can see.  We could potentially check if the tests are inside of a container and not run.

Reply:
Yep, should be mostly trivial to skip based the `$container` env. var. being set non-empty, no?